odata-protocol/11 Data Service Requests.md

```diff
@@ -1145,8 +1147,10 @@ On success, the response body MUST contain the exact count of items
 matching the request after applying any
 [`$filter`](#SystemQueryOptionfilter) or
 [`$search`](#SystemQueryOptionsearch) system query options, formatted as
-a simple primitive integer value with media type `text/plain`. Clients
-SHOULD NOT combine the system query options
+a simple primitive integer value with media type `text/plain` without `charset` parameter,
```

Comment:
I may be paranoid, but it is specified that the integer must be given in base-10 notation? (Javascript interprets `010` as octal notation for the number eight.)